src/coreclr/src/inc/cor.h

```diff
@@ -2093,7 +2093,10 @@ inline ULONG CorSigUncompressData(      // return number of bytes of that compre
 }
 
 
-extern const mdToken g_tkCorEncodeToken[];
+FORCEINLINE const mdToken (&g_tkCorEncodeTokenFn())[4] {
```

Comment:
Could you please change the signature to `mdToken CorSigDecodeTokenType(int encoded)` to make the style similar to the rest of the file?